src/main/webapp/user/User.js

```diff
@@ -30,4 +31,45 @@ class User {
   get imgKey() {
     return this._imgKey;
   }
+
+  get queryString() {
+    const query = `id=${this._id}&name=${this._name}&email=${this._email}\
+      &publicPortfolio=${this._publicPortfolio}&selfIntroduction=\
+      ${this._selfIntroduction}&imgKey=${this._imgKey}`;
+    return query;
+  }
+
+  static getUserFromQueryString() {
+    if (window.location.search != '') {
+      const GET = {};
+      const queryString = decodeURI(window.location.search.replace(/^\?/, ''));
+      queryString.split(/\&/).forEach(function(keyValuePair) {
+        const paramName = keyValuePair.replace(/=.*$/, '');
+        const paramValue = keyValuePair.replace(/^[^=]*\=/, '');
+        GET[paramName] = paramValue;
+      });
```

Comment:
This feels like it could be extracted to some "UrlQueryUtils" class so it could be re-used.

Reply:
Done.